src/buildHTML.js

```diff
@@ -66,6 +66,98 @@ export const spliceSpaces = function(children, i) {
     }
 };
 
+const thinspace = {
+    number: 3,
+    unit: "mu",
+};
+const mediumspace = {
+    number: 4,
+    unit: "mu",
+};
+const thickspace = {
+    number: 5,
+    unit: "mu",
+};
+
+export const spacings = {
```

Comment:
This all looking super clean 👍 . Since this is just data with no dependencies. Perhaps it could be extracted outside of buildHTML into it's own module? Maybe a `spacing.js`?